src/main/java/com/google/sps/youtube/YoutubeInfoScraper.java

```diff
@@ -0,0 +1,74 @@
+
+package com.google.sps.youtube;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.ChannelListResponse;
+import com.google.api.services.youtube.YouTubeRequestInitializer;
+import com.google.api.services.youtube.model.Channel;
+import com.google.api.services.youtube.model.ChannelContentDetails;
+import com.google.api.services.youtube.model.ChannelContentDetails.RelatedPlaylists;
+import java.util.List;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import com.google.sps.youtube.YoutubeResponse;
+import java.util.Properties; 
+import java.io.FileInputStream;
+
+
+public class YoutubeInfoScraper implements YoutubeResponse {
+
+  // TODO: Add seperate file to hold API Key  
+  private String API_KEY = "";
+  private static final String APPLICATION_NAME = "promotube";
+
+  private YouTube youtube;
```

Comment:
rename to youTubeClient or youTubeService

Reply:
Done.